lib/beaker/hypervisor/vagrant.rb

```diff
@@ -160,6 +160,17 @@ def cleanup
       FileUtils.rm_rf(@vagrant_path)
     end
 
+    #snapshotting depends on https://github.com/scalefactory/vagrant-multiprovider-snap
+    def take_snapshot(host,snapshot_name)
+      @logger.debug "Creating snapshot of #{host}"
```

Comment:
At the least, I should probably rename the parameter to hostname to avoid confusion.